aries-site/src/pages/components/card.mdx

```diff
@@ -51,9 +59,64 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- When presenting high-level, summarized pieces of information.
+- When presenting an item or collection of items that can best be consumed visually.
+
 ## When not to use
 
-- Do not use a Card to filter data on the same screen, instead use **\_**.
+- Do not use a Card as a clickable region to filter data on the same screen.
+- Do not use a Card to group complex content into a single container.
+- When data attributes across items needs to be analyzed, sorted, compared and filtered, a Card may not be the best tool. Instead, [DataTable](/components/datatable) may be more appropriate.
+
+<BestPracticeGroup>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'do',
+      message: 'Style filter buttons as default buttons.',
+    }}
+    scale={0.4}
+    width="100%"
+  >
+    <CardFilteringBestPractice />
+  </Example>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'dont',
+      message: `Don’t use a Card to wrap a set of buttons. Cards should act as independent regions that do not affect surrounding content.`,
+    }}
+    scale={0.4}
+    width="100%"
+  >
+    <CardFilteringBestPractice bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+<BestPracticeGroup>
```

Comment:
I wonder if this do/don't could use some text elaborating on exactly why card is not appropriate to present the content section here? If we review the criteria above, it seems to be contained, independent, individual, summarized, so I can see folks getting tripped up here.